### DISCUSSION TITLE
Pin frequently open apps and folder

### DISCUSSION THREAD

**Discussion by SIVAH-AKASH** [Flow-Launcher/Flow.Launcher#1244] [Ideas]
I would like to add a suggestion to pin certain folder and apps to the flow popup. This should of course be optional, but would be a useful functionality to have.

**Comment by jjw24** [comment]
Design wise would be similar to displaying most used results in #19. I also think this is a good one to have but both will require a bit of redesign to cater for storing the Result object, but is definitely something to add.
